internal/xds/translator/route.go

```diff
@@ -4,6 +4,7 @@ import (
 	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 	route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
 	matcher "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+	wrappers "github.com/golang/protobuf/ptypes/wrappers"
```

Comment:
`github.com/golang/protobuf` is deprecated, use `google.golang.org/protobuf` instead. (`google.golang.org/protobuf/types/known/wrapperspb` aught to be the specific package?)

Also, is there a reason to name the import instead of letting it take the default name?